scripts/move_stable.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import click
+
+# import ogr
+
+import os
+import subprocess
+from typing import List
+
+
+NAMESPACE: str = "packit"
+REPOSITORIES: List[str] = [
+    "packit",
+    "packit-service-fedmsg",
+    "packit-service-centosmsg",
+    "sandcastle",
+    "dashboard",
+    "tokman",
+    "packit-service",
+]
+# TESTING_REPOSITORY: str = "hello-world"
+STABLE_BRANCH: str = "stable"
+ROLLING_BRANCH: str = "master"
+
+
+@click.group()
+def cli() -> None:
+    pass
+
+
+@cli.command(short_help="Creates a directory with all the repositories")
+def init() -> None:
+    os.makedirs("move_stable_repositories", exist_ok=True)
+
+    for repository in REPOSITORIES:
+        subprocess.run(
+            ["git", "clone", f"github.com:{NAMESPACE}/{repository}.git"],
```

Comment:
I'm getting `Permission denied (publickey)` without the following:
```suggestion
            ["git", "clone", f"git@github.com:{NAMESPACE}/{repository}.git"],
```

Reply:
interesting, must've been my `github.com` alias in SSH config, will fix